annotation/tests/test_distribution.py

```diff
@@ -1905,3 +1924,25 @@ def test_redistribute_error(
     )
     assert exc_info.value.status_code == 500
     assert "Connection timeout" in str(exc_info.value.detail)
+
+
+def test_create_partial_validation_tasks():
+    validation_files_pages = {1: list(range(1, 81))}
+    validation_tasks = []
+    with patch(
+        "annotation.distribution.main.filter_validation_files_pages",
+        return_value=validation_files_pages,
+    ), patch("annotation.distribution.main.SPLIT_MULTIPAGE_DOC", True):
+        create_partial_validation_tasks(
+            {},
+            [{"user_id": "user1"}, {"user_id": "user2"}],
+            {},
+            1,
+            validation_tasks,
+            TaskStatusEnumSchema.pending,
+            None,
```

Comment:
This is a bool value, we should indicate this within the test, even if it is not utilized anywhere else except the mocked function. Pass in either True or False